packages/edit-site/src/components/sidebar-navigation-screen/index.js

```diff
@@ -61,14 +61,17 @@ export default function SidebarNavigationScreen( {
 						<NavigatorToParentButton
 							as={ SidebarButton }
 							icon={ isRTL() ? chevronRight : chevronLeft }
-							aria-label={ __( 'Back' ) }
+							label={ __( 'Back' ) }
+							showTooltip={ false }
 						/>
 					) }
 					{ ! isRoot && backPath && (
 						<SidebarButton
+							as={ SidebarButton }
```

Comment:
Is this right - the component is already a `SidebarButton`?

Reply:
> Is this right

Nope 🙈